Python/Fb_Request_Accepter/README.md

```diff
@@ -0,0 +1,33 @@
+# Facebook Request Accepter
+
+Script will accepts all the Friend requests on Your Facebook Account
```

Comment:
Replace with `The Script will accept all your Friend Requests on your Facebook Account`